packages/ui/Layout/Cell.js

```diff
@@ -173,13 +180,13 @@ const CellBase = kind({
 		}
 	},
 
-	render: ({component: Component, componentCss, componentRef, ...rest}) => {
+	render: ({component: Component, componentCss, componentRef, key, ...rest}) => {
 		delete rest.align;
 		delete rest.grow;
 		delete rest.shrink;
 		delete rest.size;
 
-		return <Component css={componentCss} ref={componentRef} {...rest} />;
+		return <Component css={componentCss} ref={componentRef} key={key} {...rest} />;
```

Comment:
I think instead of adding docs of key, we can just destructure rest props.
```suggestion
		const {key, ...restProps} = {...rest};

		return <Component css={componentCss} key={key} ref={componentRef} {...restProps} />;
```